static-assets/components/cstudio-common/common-api.js

```diff
@@ -2566,16 +2566,24 @@ var nodeOpen = false,
                       success: function () {
                         var pasteCb = {
                           success: function (pasteResponse) {
+                            opCallBack.success();
```

Comment:
Moved the success callback call here since it's a success for duplication, not for saving of the form that opens after duplicating.